### PR TITLE
chore: add azure node manager image v0.7.0

### PR DESCRIPTION
**Pull request by nilo19**
chore: add azure node manager image v0.7.0

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -658,6 +658,15 @@ for CSI_NODE_DRIVER_REGISTRAR_VERSION in ${CSI_NODE_DRIVER_REGISTRAR_VERSIONS};
   echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
 done
 
+AZURE_CLOUD_NODE_MANAGER_VERSIONS="
+0.7.0
```
Comment (feiskyer) [inline vhdbuilder/packer/install-dependencies.sh:662]: should also add 0.6.0 (for k8s v1.19)

Comment (nilo19) [reply vhdbuilder/packer/install-dependencies.sh:662]: ok, so what about 1.18? do we support?

File: vhdbuilder/packer/test/vhd-content-test.ps1 [hunk 2]
```diff
@@ -169,7 +169,8 @@ function Test-ImagesPulled
                 "mcr.microsoft.com/oss/kubernetes/pause:1.4.0",
                 "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.0.1-alpha.1-windows-1809-amd64",
                 "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v1.2.1-alpha.1-windows-1809-amd64",
-                "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.0.1")
+                "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.0.1",
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v0.7.0")
```
Comment (feiskyer) [inline vhdbuilder/packer/test/vhd-content-test.ps1:173]: same here

File: vhdbuilder/packer/install-dependencies.sh (hunk 1 above)
Comment (feiskyer) [reply vhdbuilder/packer/install-dependencies.sh:663]: Good point. Should also add `v0.5.1`.

**Pull request closed without merging**